dot/parachain/provisioner/test_helper.go

```diff
@@ -73,11 +73,11 @@ func signedBitfield(
 	require.NoError(t, err)
 
 	publicKeyBytes := keyPair.Public().Encode()
-	validatorID := parachaintypes.ValidatorID(publicKeyBytes)
+	ValidatorPublicKey := parachaintypes.ValidatorPublicKey(publicKeyBytes)
```

Comment:
Let's make the variable unexported.
```suggestion
	validatorPublicKey := parachaintypes.ValidatorPublicKey(publicKeyBytes)
```